locales/da_DK.js

```diff
@@ -0,0 +1,14 @@
+export default {
+    locale_code: 'da-DK',
```

Comment:
Why have locale code inside the locale and why da-DK instead of da_DK? We use ll_CC everywhere.

Reply:
This is to avoid replacing - to _ everytime we use built in functions eg: date formatters uses `da-DK` format, but we can change this to _ for uniformity 😄